### ISSUE TITLE
possible bug in set_trainable()? 

### ISSUE THREAD

**Issue by FernandoCamaro** [multimodallearning/pytorch-mask-rcnn#45]
Following the coco.py, at the beginning, for training only the heads, the set_trainable function will make all the params not belonging to the heads with  requires_grad = False, which is fine, but later when for example we continue the training with the intention to update all the layers, set_trainable will not set the  requires_grad property of all params to True; according to code it only ensures that the params not intended to be trained will have requires_grad = False...
